### PR TITLE
Add properties projects.to_feature, for projects.json API endpoint

### PR DESCRIPTION
**Pull request by mikelmaron**

### CODE REVIEW

File: osmtm/models.py [hunk 1]
```diff
@@ -575,6 +575,19 @@ def to_feature(self):
...
+        properties['per_task_instructins'] = self.per_task_instructions
+        properties['status'] = self.status
+        properties['created'] = self.created.strftime('%FT%TZ')
+        if self.author:
+            properties['author'] = self.author.username
+        properties['last_update'] = self.last_update.strftime('%FT%TZ')
+        properties['license'] = self.license_id
+        properties['priority'] = self.priority
+        properties['done'] = self.done
+        properties['vallidated'] = self.validated
```
Comment (pgiraud) [inline osmtm/models.py]: Typo here: 2 "l"s.

File: osmtm/models.py [hunk 2]
```diff
@@ -575,6 +575,19 @@ def to_feature(self):
         properties = {}
         properties['name'] = self.name
         properties['description'] = self.description
+        properties['short_description'] = self.short_description
+        properties['instructions'] = self.instructions
+        properties['per_task_instructins'] = self.per_task_instructions
```
Comment (pgiraud) [inline osmtm/models.py]: Typo here: missing "o".

**Comment by pgiraud** [comment]
@mikelmaron can you please squash the commits into only one and prevent the history from showing a merge commit?

**Comment by mikelmaron** [comment]
@pgiraud any tips on squashing ... try and failing

**Comment by pgiraud** [comment]
Duplicate of #576.

**Pull request closed without merging**